docs/crio.conf.5.md

```diff
@@ -278,11 +278,43 @@ The "crio.runtime.runtimes" table defines a list of OCI compatible runtimes.  Th
 
 **allowed_annotations**=[]
   A list of experimental annotations this runtime handler is allowed to process.
+  This field is currently DEPRECATED. If you'd like to use allowed_annotations, please use a workload.
```

Comment:
I don't know how it's done else where, but I'd be tempted to move this line about line 279 and bold it or perhaps put it behind 3 or 4 hash tags so it's hard to miss